rpc/common/service.proto

```diff
@@ -63,27 +63,39 @@ message Misconfiguration {
 }
 
 message MisconfResult {
-  string namespace = 1;
-  string message   = 2;
-  string type      = 3;
-  string id        = 4;
-  string title     = 5;
-  string severity  = 6;
+  string namespace               = 1;
+  string         message         = 2;
+  PolicyMetadata policy_metadata = 3;
+  CauseMetadata  cause_metadata  = 4;
```

Comment:
We should not reuse the field number for some reasons. It leads to ambiguity.
https://protobuf.dev/programming-guides/proto3/#assigning